Deliverable 1/API Examples/reportMetric.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Component_Request": {
+    "example 1": {
+      "component": "querying",
+      "label": "Query Time",
+      "value": 1464
+    },
+    "example 2": {
+      "component": "crawling",
+      "label": "Broken Link",
+      "value": {
+        "broken_link": "abc.com",
+        "last_page": "xyz.com",
+        "origin_page": "rpi.edu"
```

Comment:
Second example is to show that the value can be different formats for different labels